package.json

```diff
@@ -2,7 +2,7 @@
     "name": "sdfv",
     "displayName": "DaCe SDFG Viewer (SDFV)",
     "description": "DaCe SDFG Viewer (SDFV) for VS Code",
-    "version": "0.3.0",
+    "version": "0.3.1",
```

Comment:
aren't we 0.3.2?

Reply:
We are after this, this is a quirk with github actions that I'm still trying to find a solution to - when using actions to patch the plugin, the version number of package.json should be what you are PRE-patch. So if trying to patch from 0.3.1 to 0.3.2, your package.json should read 0.3.1, counterintuitively..